src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_commands.py

```diff
@@ -128,6 +128,16 @@ def load_flexibleserver_command_table(self, _):
                                  custom_func_name='flexible_firewall_rule_update_custom_func',
                                  custom_func_type=flexible_server_custom_common)
 
+    with self.command_group('postgres flexible-server migration', postgres_flexible_firewall_rule_sdk,
+                            custom_command_type=flexible_servers_custom_postgres,
+                            client_factory=cf_postgres_flexible_firewall_rules,
+                            is_experimental=True) as g:
+        g.custom_command('create', 'migration_create_func', custom_command_type=flexible_server_custom_common)
+        g.custom_show_command('show', 'migration_show_func', custom_command_type=flexible_server_custom_common)
+        g.custom_command('list', 'migration_list_func', custom_command_type=flexible_server_custom_common)
+        g.custom_command('update', 'migration_update_func', custom_command_type=flexible_server_custom_common)
+        g.custom_command('delete', 'migration_delete_func', custom_command_type=flexible_server_custom_common)
```

Comment:
do you need to add confirmation for delete operation?
https://github.com/Azure/azure-cli/blob/fb60a21d0abaa533e36fbec0974b59bae4f37d75/src/azure-cli/azure/cli/command_modules/storage/commands.py#L108